File: KnowledgeSharingApi/KnowledgeSharingApi.Domain/Models/Entities/Entities/Conversation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace KnowledgeSharingApi.Domains.Models.Entities.Entities
{
    public class Conversation : Entity
    {
        [Key]
        public Guid ConversationId { get; set; } = Guid.Empty;
        public string ConversationName { get; set; } = string.Empty;
        public string? Thumbnail { get; set; }
        protected override Conversation Init()
        {
            return new Conversation();
        }
    }
}

File: KnowledgeSharingVue/knowledge-sharing-vue/src/js/models/entities/conversation.js

import Entity from "../entity";

class Conversation extends Entity {
    constructor() {
        super();
        this.ConversationId = null;
        this.ConversationName = null;
        this.Thumbnail = null;
    }
    
    init() {
        return new Conversation();
    }
}

export default Conversation;
